test/loader/test_link_neighbor_loader.py

```diff
@@ -213,14 +212,12 @@ def test_temporal_heterogeneous_link_neighbor_loader():
         edge_label_time=edge_time,
         batch_size=32,
         time_attr='time',
-        neg_sampling_ratio=0.5,
+        neg_sampling_ratio=1.0,
         drop_last=True,
     )
     for batch in loader:
         # Check if each seed edge has a different batch:
-        assert int(batch['paper'].batch.max()) + 1 == 32 + 16
-        # Check if each seed edge has a different source and dstination node:
-        assert batch['paper'].num_nodes >= 2 * (32 + 16)
+        assert int(batch['paper'].batch.max()) + 1 == 32
```

Comment:
Why isn't this 64? Won't the negative samples also have a disjoint batch?

Reply:
I personally think it makes more sense to group the negatives to its own batch, in particular because they now share time information.